### PR TITLE
[Job Launcher] Rework webhook service - repository schema

### PR DESCRIPTION
**Pull request by flopez7**
## Description

Rework service - repository schema in webhook module

## Summary of changes

Remove all unnecessary methods from user repository and extend it from base Repository

## How test the changes

`yarn test
`
## Related issues

#1532

### CODE REVIEW

**Review by portuu3 — APPROVED** [review]

File: packages/apps/job-launcher/server/src/modules/webhook/webhook.repository.ts [hunk 1]
```diff
@@ -1,77 +1,78 @@
...
-    const webhookEntity = await this.webhookEntityRepository.findOne({
-      where,
-      ...options,
-    });
+  public async updateOne(
+    id: number,
+    webhook: Partial<WebhookEntity>,
+  ): Promise<WebhookEntity | null> {
+    try {
+      const result = await super.update(id, webhook);
```
Comment (Dzeranov) [inline packages/apps/job-launcher/server/src/modules/webhook/webhook.repository.ts:44]: Should it be `super.update()` or `this.update()`

File: packages/apps/job-launcher/server/src/modules/webhook/webhook.repository.ts [hunk 2]
```diff
@@ -1,77 +1,78 @@
...
+      console.log(webhook);
```
Comment (Dzeranov) [inline packages/apps/job-launcher/server/src/modules/webhook/webhook.repository.ts:45]: Please, remove `console.log()`

**Review by Dzeranov — DISMISSED** [review]
lgtm
@portuu3 please take a look

**Review by Dzeranov — DISMISSED** [review]
lgtm

File: packages/apps/job-launcher/server/src/modules/cron-job/cron-job.service.ts
Comment (portuu3) [inline packages/apps/job-launcher/server/src/modules/cron-job/cron-job.service.ts:1]: @Dzeranov `CronJobService` should use `WebhookService` to access the webhook data, not directly `WebhookRepository` right?

Comment (Dzeranov) [reply packages/apps/job-launcher/server/src/modules/cron-job/cron-job.service.ts:1]: It's ok to directly use repositories.

File: packages/apps/job-launcher/server/src/modules/webhook/webhook.entity.ts [hunk 3]
```diff
@@ -34,14 +34,14 @@ export class WebhookEntity extends BaseEntity {
...
   @Column({ type: 'timestamptz' })
-  public waitUntil: Date;
+  public waitUntil: Date = new Date();
 
   @Column({
     type: 'enum',
     enum: WebhookStatus,
   })
-  public status: WebhookStatus;
+  public status: WebhookStatus = WebhookStatus.PENDING;
```
[the remark this answers is not in the document]
Comment (eugenvoronov) [reply packages/apps/job-launcher/server/src/modules/webhook/webhook.entity.ts:46]: Yeah I see it in their doc, thanks @flopez7

**Review by portuu3 — APPROVED** [review]

**Pull request merged**